### ISSUE TITLE
Can't SIGTERM/SIGSTOP to cancel builds anymore

### ISSUE THREAD

**Issue by mjmasn** [meteor/meteor#8994]
Not a life-threatening issue but it seems lately that `Crtl+C` (`SIGTERM`) after running `meteor` no longer has the ability to kill the process while the app is building. 

Can't be sure when this first started happening but I've noticed it increasingly lately - certainly somewhere around the 1.5 mark.

`Ctrl+Z` (`SIGSTOP`) also shares this problem, although`Ctrl+\` (`SIGQUIT`) does work.

Below the warning appeared immediately so I knew I needed to cancel the build and install various dependencies but it took ~2 or 3 minutes before I could actually cancel. The previous behaviour was an immediate exit on `SIGTERM`.

Platform: **maxOS Sierra 10.12.5**, **Meteor 1.6-beta.2** (I've experience this with 1.5 as well)
```
=> Started proxy.
Warning: unable to resolve "transform-class-properties" in plugins of /home/xxx/.babelrc
^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
Unable to resolve some modules:

  "react" in /home/xxx/imports/client/core/setup/modules.js
(web.browser)
  "react-i18next" in
/home/xxx/imports/client/core/setup/modules.js (web.browser)
  "react-mounter" in
/home/xxx/imports/client/core/setup/modules.js (web.browser)
  "prop-types" in
/home/xxx/imports/client/core/setup/modules.js (web.browser)
  "react-dom" in /home/xxx/imports/client/core/setup/modules.js
(web.browser)

If you notice problems related to these missing modules, consider running:

  meteor npm install --save react react-i18next react-mounter prop-types react-dom
```

**Comment by abernix** [comment]
Is there something else that has changed in your environment which could be causing it?  I haven't noticed any difference in this behavior and I certainly do this dozens of times per day.

> Can't be sure when this first started happening but I've noticed it increasingly lately - certainly somewhere around the 1.5 mark.

Since you're experiencing the problem, could you try different Meteor versions and report when it does happen?  It makes the different between having no idea what could have caused a change and having some idea. 🙂

**Comment by mjmasn** [comment]
@abernix I realised afterwards that I've experienced the issue on a Linux VM as well (Ubuntu 16.04 running in VirtualBox on OSX). Will have to do some more digging. The only common connection I can see is that both were in iTerm2 (I was using the VM via SSH), but `Ctrl+C` works fine elsewhere so it seems unlikely to be the cause. I suspect it's probably a package I'm using - the only build-related ones are`stylus` and `juliancwirko:postcss` so I'll start there when I have time. 

I'm moving country at the end of this week but I've set a calendar reminder to take another look on 21st August.
